style.css

```diff
@@ -0,0 +1,94 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  font-size: 16px;
+  color: #333;
+  background-color: #fff;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  overflow-x: hidden;
+  overflow-y: auto;
+  width: 450px;
+  display: flex;
+  flex-direction: column;
+}
+
+nav {
+  background-color: #333;
+  padding: 20px;
+  color: #fff;
+  position: feixed;
+  width: 450px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.header .logo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 4px;
+  min-width: 100px;
+  width: 100%;
+}
+
+i {
+  display: flex;
+  justify-items: flex-end;
+}
+
+.container {
+  background-color: #1c1a19;
+  display: flex;
+  justify-content: space-between;
+  align-self: center;
+  flex-wrap: wrap;
+  height: 580px;
+  width: 450px;
+  left: 0;
+  top: 92px;
+  border-radius: 0;
+  padding: 100px 10px 10px 14px;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+}
+
+.row {
+  margin-top: 8px;
+}
+
+h1 {
+  color: #ff6b00;
+  font-size: 30px;
+  font-weight: bold;
+  padding: 8px;
+}
```

Comment:
- [ ] As part of the requirement for this milestone is to make sure your website looks like the given design, I think your text sizes are smaller than the Figma design, I'll suggest you go through the design and adjust your sizes appropriately. :smile: :+1:  

> Your website should look exactly like the design (e.g. font, colors, images, text, margins)

- YOUR CURRENT DESIGN:
![Screenshot from 2023-05-03 17-52-36](https://user-images.githubusercontent.com/107632780/235987598-44cc1517-92d4-40fd-b4ec-8734a77e0685.png)

- FIGMA DESIGN:
![Screenshot from 2023-05-03 17-52-57](https://user-images.githubusercontent.com/107632780/235988906-0f750007-e86b-4604-a552-1a91fda22bb1.png)